warehouse/models/mart/payments/fct_payments_rides_v2.sql

```diff
@@ -0,0 +1,344 @@
+{{ config(
+    post_hook=[
+" {{ create_row_access_policy(
+    filter_column = 'participant_id',
+    filter_value = 'mst',
+    principals = ['serviceAccount:mst-payments-user@cal-itp-data-infra.iam.gserviceaccount.com']
+) }}",
+" {{ create_row_access_policy(
+    filter_column = 'participant_id',
+    filter_value = 'sacrt',
+    principals = ['serviceAccount:sacrt-payments-user@cal-itp-data-infra.iam.gserviceaccount.com']
+) }}",
+" {{ create_row_access_policy(
+    filter_column = 'participant_id',
+    filter_value = 'sbmtd',
+    principals = ['serviceAccount:sbmtd-payments-user@cal-itp-data-infra.iam.gserviceaccount.com']
+) }}",
+" {{ create_row_access_policy(
+    filter_column = 'participant_id',
+    filter_value = 'clean-air-express',
+    principals = ['serviceAccount:clean-air-payments-user@cal-itp-data-infra.iam.gserviceaccount.com']
+) }}",
+" {{ create_row_access_policy(
+    principals = ['serviceAccount:metabase@cal-itp-data-infra.iam.gserviceaccount.com',
```

Comment:
This kind of list feels like it would be best to maintain somewhere outside code and feed in programmatically, rather than living in explicit form in SQL. Not sure if it's worth taking any action on this PR, though, since that's more of a design + process conversation that touches on offboarding procedures and so on.

Reply:
Right I think the ideal world would be managing these in terraform and providing a structured dictionary for the macro to handle.